Porting.md

```diff
@@ -152,6 +167,8 @@ mka hybris-boot
 mka systemimage
 ```
 
+Here is also recomended to set `-j[num]` to do parallel building, this reduces build time. *NOTE* this is not needed on halium-7.1 since it will calculate parallel building based on CPU performance.
```

Comment:
`mka -jX` is completely useless as the mka tool automatically select the best number of threads. And mka is available both and 14.1/7.1 and in 12.1/5.1. Basically `mka` is `make -j<threads>`